frontend/test/cypress/integration/product/add_to_cart.cy.ts

```diff
@@ -3,132 +3,111 @@ describe('product page add to cart', () => {
       cy.loadProductPageByPath(
          '/products/spudger-retail-3-pack?variantid=39419992408154'
       );
-      cy.findByTestId('cart-drawer-open').click().wait(10);
-      cy
-         .findAllByTestId('cart-drawer-item-count')
+      cy.findByTestId('cart-drawer-open').click();
+      cy.findAllByTestId('cart-drawer-item-count')
          .should('exist')
          .invoke('text')
          .should('eq', '0');
       cy.findByTestId('cart-drawer-quantity').should('not.exist');
-      cy.findByTestId('cart-drawer-close').click().wait(10);
+      cy.findByTestId('cart-drawer-close').click();
```

Comment:
I have my reservations over this change, mainly because these were implemented to cut down on flakiness that i found https://github.com/iFixit/react-commerce/pull/854/commits/243c9f73dbf1b0c107f680e3028bfd3a818d796a

But if something else fixed this than im glad we could reduce the gross `wait()`s everywhere

Reply:
Hm yea I'm not sure why the behavior is different all of a sudden. Maybe we changed some animations at some point.

I'm not aware of a way to stress test in cypress. It would be nice to run these tests many times in parallel to try to flesh out any flakiness.

---

Found some docs about it https://docs.cypress.io/guides/guides/parallelization#Overview

> While parallel tests can also technically run on a single machine, we do not recommend it since this machine would require significant resources to run your tests efficiently.